src/cozmo/util.py

```diff
@@ -134,6 +134,17 @@ def degrees(self):
         '''float: The angle in degrees.'''
         return self._radians / math.pi * 180
 
+    @property
+    def abs_value(self):
+        """:class:`cozmo.util.Angle`: The absolute value of the angle.
+        
+        If the Angle is positive then it just returns the Angle, otherwise it
+        returns -Angle.
+        """
+        if self.__lt__(radians(0)):
+            return radians(0) - self
+        return self
```

Comment:
We might want to do `return Angle(radians=self._radians)` here, so that it always returns a new object (not just if it's negative) - otherwise the behavior of e.g.

```python
angle = angle.abs_value()
angle += other_angle
```
varies depending on if the angle was negative or not (negative will return a copy, so this wouldn't change the original, but positive angles would return a reference and would stomp the value it was pointing at).